spec/rails_helper.rb

```diff
@@ -64,8 +64,16 @@
 
   # start by truncating all the tables but then use the faster transaction strategy the rest of the time.
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:deletion)
+  end
+
+  config.before(:each) do
     DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.start
+  end
+
+  config.after(:each) do
+    DatabaseCleaner.clean
```

Comment:
@NoyaArie 
I've configured DatabasCleaner to delete all DB before all, and clean after each test, checked it with some puts seems to work as intended...